Create pavanh007

pavanh007 opened the pull request:
Required changes are updated: PR #6714

<!-- If your PR fixes an open issue, use `Closes #999` to link your PR with the issue. #999 stands for the issue number you are fixing -->

## Fixes Issue

<!-- Remove this section if not applicable -->

<!-- Example: Closes #31 -->

## Changes proposed

<!-- List all the proposed changes in your PR -->

<!-- Mark all the applicable boxes. To mark the box as done follow the following conventions -->
<!--
[x] - Correct; marked as done
[X] - Correct; marked as done

[ ] - Not correct; marked as **not** done
-->

## Check List (Check all the applicable boxes) <!-- Follow the above conventions to check the box -->

- [ ] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] All new and existing tests passed.
- [ ] This PR does not contain plagiarized content.
- [ ] The title of my pull request is a short description of the requested changes.

## Screenshots

<!-- Add all the screenshots which support your changes -->

## Note to reviewers

<!-- Add notes to reviewers if applicable -->


<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/6738"><img src="https://gitpod.io/button/open-in-gitpod.svg"/></a>

khushalarora reviewed (changes requested):
Hey @pavanh007, thank you for contributing :tada: 
Please rename your file to **`data/pavanh007.json`** :+1:

pavanh007 commented:
> Hey @pavanh007, thank you for contributing 🎉 Please rename your file to **`data/pavanh007.json`** 👍

can you suggest me

SurajPratap10 commented:
> > Hey @pavanh007, thank you for contributing 🎉 Please rename your file to **`data/pavanh007.json`** 👍
> 
> can you suggest me

There's a `data` folder in the repo where every user's data is kept, you have to put your file under that folder.

pavanh007 commented:
> > > Hey @pavanh007, thank you for contributing 🎉 Please rename your file to **`data/pavanh007.json`** 👍
> > 
> > 
> > can you suggest me
> 
> There's a `data` folder in the repo where every user's data is kept, you have to put your file under that folder.

>> As you suggest, I removed existing file and added in data folder

kumarsonsoff3 reviewed (dismissed):
Please add the file extension `.json` in the end of your [filename here](https://github.com/pavanh007/LinkFree/edit/pavanh007-json/data/pavanh007).

Let us know if you need any help with this! :)

Vishrut19 reviewed (changes requested):
Please change the filename to data/pavanh007.json

SaraJaoude commented:
> Please change the filename to data/pavanh007.json

@Vishrut19 we have brought to your attention a number of times not to duplicate comments.  As we have highlighted to you this causes confusion to the author, maintainers - and can also scare off contributors. If you do not have time to be looking through historical comments then perhaps you should not be reviewing PRs.

Vishrut19 commented:
> > Please change the filename to data/pavanh007.json
> 
> @Vishrut19 we have brought to your attention a number of times not to duplicate comments. As we have highlighted to you this causes confusion to the author, maintainers - and can also scare off contributors. If you do not have time to be looking through historical comments then perhaps you should not be reviewing PRs.

Hey Sara! First of I am extremely Sorry for duplicate comments but I tried to make pavan understand that this is the naming convention that needs to be followed i.e. `data/<yourname>/json`. I thought putting a comment like that would make him understand about the naming convention along with the file format. Hope it makes sense.

Vishrut19 commented on data/pavanh007:
```diff
@@ -0,0 +1,34 @@
+{
+    "name": "Pavan H",
+    "type": "personal",
+    "bio": "Backend developer" | "Tech enthusiast",
```
The format is incorrect. The correct format is "Backend developer | Tech enthusiast"

Pradumnasaraf reviewed (approved):
Hey everyone,

Thank you all for the review. Since the pull request has been open for a while, I will be merging it into a temporary branch to address any existing errors before merging it into the main branch.
